sdk/communication/communication-sms/test/smsClient.spec.ts

```diff
@@ -2,54 +2,105 @@
 // Licensed under the MIT license.
 
 import { assert } from "chai";
-import { SmsClient, SendRequest } from "../src/smsClient";
-import { record, Recorder, env, RecorderEnvironmentSetup } from "@azure/test-utils-recorder";
+import { SmsClient, SmsSendRequest, SmsSendOptions } from "../src/smsClient";
+import { env, isPlaybackMode, record, Recorder } from "@azure/test-utils-recorder";
 import { isNode } from "@azure/core-http";
 import * as dotenv from "dotenv";
+import * as sinon from "sinon";
+import { Uuid } from "../src/utils/uuid";
+import { recorderConfiguration } from "./utils/recordedClient";
 
 if (isNode) {
   dotenv.config();
 }
 
-const recorderConfiguration: RecorderEnvironmentSetup = {
-  replaceableVariables: {
-    AZURE_COMMUNICATION_LIVETEST_CONNECTION_STRING: "endpoint=https://endpoint/;accesskey=banana",
-    AZURE_PHONE_NUMBER: "+18005551234"
-  },
-  customizationsOnRecordings: [
-    (recording: string): string => recording.replace(/(https:\/\/)([^/',]*)/, "$1endpoint"),
-    (recording: string): string =>
-      recording.replace(/"messageId"\s?:\s?"[^"]*"/g, `"messageId":"Sanitized"`)
-  ],
-  queryParametersToSkip: []
-};
-
 describe("SmsClient", async () => {
   let recorder: Recorder;
 
   beforeEach(async function() {
     recorder = record(this, recorderConfiguration);
+    if (isPlaybackMode()) {
+      sinon.stub(Uuid, "generateUuid").returns("sanitized");
+      sinon.stub(Date, "now").returns(0);
+    }
   });
 
   afterEach(async function() {
     if (!this.currentTest?.isPending()) {
       await recorder.stop();
     }
+    if (isPlaybackMode()) {
+      sinon.restore();
+    }
+  });
+
+  it("sends a SMS message to multiple recipients", async () => {
+    const connectionString = env.AZURE_COMMUNICATION_LIVETEST_CONNECTION_STRING as string;
+    const fromNumber = env.AZURE_PHONE_NUMBER as string;
+    const validToNumber = env.AZURE_PHONE_NUMBER as string;
+    const invalidToNumber = "+18332321226444";
+
+    const recipients = [validToNumber, invalidToNumber];
```

Comment:
Might need to add a few more test cases for two valid numbers, etc. But can be addressed in separate PR